### DISCUSSION TITLE
Command Line Multiple Copies Behavior

### DISCUSSION THREAD

**Discussion by kevhua** [sumatrapdfreader/sumatrapdf#2349] [Q&A]
Hello!
I'm sorry first of all if this question has already been asked before - I've tried glancing around but haven't personally been able to find an answer, so thought to just ask it here. I've gotten the command line print functionality to work fine on Win11 

`> SumatraPDF.exe -silent -print-to-default -print-settings "3x" filename.pdf`

But I've noticed that the way it prints is a bit.. staggered. Rather than printing out 3 copies back to back, it prints out a single copy, quiets down for a few seconds, then prints the next, repeat. Is this normal, or is there a way to just have it do the printing all at once (like selecting multiple copies in most applications when printing)? I will mention that I am using Python 3.10 to make the call via the os library, but I don't believe that should have any impact. 

Thank you for taking the time to read this!
Best,
Kevin

**Comment by kjk** [comment]
When you specify `-print-settings "3x"` Sumatra will  set `dmCopies` to 3 for the printer driver.

What printer driver does with it is an implementation detail of the printer driver. It depends on the printer.

In other words: only the company that wrote the printer driver will know for sure.

**Reply by kevhua** [reply]
Ahh, that makes sense.. If it is based on that, then I suppose there isn't really anything that I can do about that. For reference then to anyone who might be interested, Brother's DCP-L2550DW exhibits that "one copy at a time" behavior. I don't suppose there's an alternative method of specifying copies in such a way that it prints everything all at once rather than one by one other than generating a new PDF file consisting of the the original merged n times?

Thank you for the quick clarification by the way, super helpful :)
